--- a/katana_public_api_client/generated/api/purchase_order/receive_purchase_order.py
+++ b/katana_public_api_client/generated/api/purchase_order/receive_purchase_order.py
@@ -32,7 +32,7 @@ def _build_response(
         "url": "/purchase_order_receive",
     }

-    _kwargs["json"]: dict[str, Any] | list[dict[str, Any]]
+    _kwargs["json"] = None  # type: dict[str, Any] | list[dict[str, Any]]
     if isinstance(body, list):
         _kwargs["json"] = []
         for componentsschemas_purchase_order_receive_request_type_0_item_data in body:
